Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -1533,25 +1533,28 @@ sendSmsToDriver merchantShortId opCity driverId volunteerId _req@SendSmsReq {..}
       sender = smsCfg.sender
   withLogTag ("personId_" <> personId.getId) $ do
     case channel of
-      SMS -> whenJust messageKey $ \mkey -> do
+      SMS -> do
+        mkey <- fromMaybeM (InvalidRequest "Message Key field is required for channel : SMS") messageKey --whenJust messageKey $ \mkey -> do
```

Comment:
commented code

Reply:
I have commented the previous code.. for the case if we want to revert to the previous change while refactoring the error codes for solving 4xx

Reply:
Will remove this commented code, once refactoring is done